src/FixedDataTable.js

```diff
@@ -1244,7 +1261,7 @@ class FixedDataTable extends React.Component {
     }
 
     if (scrollYChanged && onVerticalScroll) {
-      onVerticalScroll(scrollY);
+      onVerticalScroll(scrollY, firstRowIndex, endRowIndex);
```

Comment:
Here `firstRowIndex` and `endRowIndex` refers to latest row indexes with correspond to `scrollY`. So all's good.